### ISSUE TITLE
Create new release to fix target API warning

### ISSUE THREAD

**Issue by ltsSmitty** [OpenRCT2/plugin-devtools#6]
The current codebase has a targetApiVersion noted, but a new version hasn't been released since the changes were implemented, leading to this warning on each boot:

`Plug-in “DevTools” does not specify a target API version or specifies it incorrectly. Emulating deprecated APIs.`.

**Comment by AaronVanGeffen** [comment]
Indeed, this issue has already been addressed by #5. We really should tag a new release for this plugin.

**Comment by AaronVanGeffen** [comment]
Fixed as of [v1.2](https://github.com/OpenRCT2/plugin-devtools/releases/tag/v1.2).
